url.bs

```diff
@@ -85,6 +85,170 @@ valid input. User agents, especially conformance checkers, are encouraged to rep
  unclear to other developers.
 </div>
 
+<table>
+ <thead>
+  <tr>
+   <th>Error Code
+   <th>Error Message
+ <tbody>
+  <tr>
+   <td><dfn>unexpected-c0-control-or-space</dfn>
+   <td>This error occurs when the input to the URL parser contains leading or trailing
```

Comment:
If we make the header "Description" as you suggest we could make this "The input to the URL parser ..." and not have the repeating introductory phrase.